README.md

```diff
@@ -1000,7 +1000,6 @@ Multiplayer game servers and [browser games](https://en.wikipedia.org/wiki/Brows
 - [Net64+](https://net64-mod.github.io) `⚠` - Net64 aka SM64O allows playing Super Mario 64 in an online multiplayer mode. Net64+ is the official continuation of the program and features an integrated server list. ([Source Code](https://github.com/Tarnadas/net64plus-server/), [Clients](https://github.com/Tarnadas/net64plus/)) `MIT` `Nodejs`
 - [node-virtual-gamepads](https://github.com/jehervy/node-virtual-gamepads) - Turn your smartphone into a game controller, keyboard, or touchpad for a remote Linux OS machine. `MIT` `Nodejs/CoffeScript`
 - [piqueserver](https://github.com/piqueserver/piqueserver) - Server for openspades, the first-person shooter in a destructible voxel world. ([Clients](https://github.com/yvt/openspades)) `GPL-3.0` `Python/C++`
-- [Posio](https://github.com/abrenaut/posio) - Geography multiplayer game. `MIT` `Python`
```

Comment:
By the look of the demo, the very simple feature set, very small code base and absence of issues, I'd say this does not need much maintenance and could be kept on the list. @abrenaut do you still use/maintain Posio?

```
$ cloc posio/
      24 text files.
      24 unique files.                              
       7 files ignored.

github.com/AlDanial/cloc v 1.86  T=0.03 s (625.7 files/s, 46204.5 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
Python                           8            133             89            331
JavaScript                       1            104             98            180
CSS                              2             32              0            172
YAML                             2              4              3             76
HTML                             2             18              6             69
Markdown                         2             23              0             31
JSON                             1              0              0             21
Dockerfile                       1              6              1              6
-------------------------------------------------------------------------------
SUM:                            19            320            197            886
-------------------------------------------------------------------------------
```

Reply:
Hi @nodiscc , I'm still maintaining it but not adding any new feature. Thanks for checking!